### PR TITLE
add rm tool

### PR DESCRIPTION
**Pull request by calvinmetcalf**
for use when you do something stupid like add the wrong name to the users.json and add that name to all 12 different repos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,14 @@ Would add all users in `example-users.json` as owners to the
 
 `ownership-all` doesn't take any arguments: it adds all users in
 `example-users.json` to all packages in `example-packages.json`.
+
+## ownership-rm
+
+`ownership-rm` takes aone argument, the name of a user, and removes
```
Comment (tmcw) [inline README.md]: sp - here `aone` should be `one`

**Comment by tmcw** [comment]
Awesome - noted one spelling issue, otherwise good to go.

**Comment by calvinmetcalf** [comment]
fixed

**Comment by tmcw** [comment]
Thanks!

**Pull request merged**